ArduCopter/system.cpp

```diff
@@ -146,11 +146,15 @@ void Copter::init_ardupilot()
     barometer.set_log_baro_bit(MASK_LOG_IMU);
     barometer.calibrate();
 
+#if RANGEFINDER_ENABLED == ENABLED
     // initialise rangefinder
     init_rangefinder();
+#endif
 
+#if HAL_PROXIMITY_ENABLED
```

Comment:
I contend that the `init_proximity` wrapper here does nothing and we should just make the call on g2:

```
// initialise proximity sensor
void Copter::init_proximity(void)
{
#if HAL_PROXIMITY_ENABLED
    g2.proximity.init();
#endif
}
```

Reply:
The decision to access other initialization methods is made by this method.
I want to match the style of this method.